ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml

```
@@ -0,0 +1,108 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: openshift-pipelines-runner
+releases:
+  latest:
+    release:
+      channel: candidate
+      version: "4.17"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: openshift-pipelines-serverless-rosa-classic-iib
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      AWS_REGION: us-east-1
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+              [
+                {"step": "openshift-pipelines-tests", "failure_type": "all", "classification": "Product to Product Test Failure", "jira_project": "SRVKP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"], "jira_epic": "!default"}
```

How about these changes? if operator-install fails, then this failures shouldn't get reported I guess, wdyt?
```suggestion
                {"step": "openshift-pipelines-tests", "failure_type": "all", "classification": "Product to Product Test Failure", "jira_project": "SRVKP", "group": {"name": "p2p-tests", "priority": 2}, "jira_additional_labels": ["!default"], "jira_epic": "!default"}
```